### DISCUSSION TITLE
Still maintained?

### DISCUSSION THREAD

**Discussion by hempels** [vitaly-t/pg-promise#830] [General]
There are a few outstanding PRs, including an update to the current release of `node-postgres` that haven't seen any activity. Just want to make sure the project is still being maintained?

Does anyone except @vitaly-t have merge privileges? This project has become a critical component for a lot of software out in the world, so ideally there is some avenue other than forking to ensure its future viability in the event that something happens to our beloved benefactor.

P.S. I'm not suggesting anything about the project. I am only asking questions.

**Comment by vitaly-t** [comment]
It is still maintained, yes, I will make time for it soon ;)
